projects/admin/src/app/record/detail-view/template-detail-view/template-detail-view.component.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2020 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { RecordService } from '@rero/ng-core';
+import { Observable, of, Subscription } from 'rxjs';
+
+@Component({
+  selector: 'admin-template-detail-view',
+  templateUrl: './template-detail-view.component.html'
+})
+export class TemplateDetailViewComponent implements OnInit, OnDestroy {
+
+  /** Observable resolving record data */
+  record$: Observable<any>;
+
+  /** Observable of the imported record in marc format */
+  marc$: Observable<any>;
+
+  /** Record subscription */
+  private _recordObs: Subscription;
```

Comment:
private properties must be placed after public ones.

Reply:
Done